### DISCUSSION TITLE
Wie und wo eine Datenbank einrichten...?

### DISCUSSION THREAD

**Discussion by frankymusik** [OpenDTU-App/opendtu-react-native#169] [Q&A]
Ich würde so gerne ein paar Graphen sehen wollen, das, was in den Wechselrichter reingeht, den Tag über, aber dazu brauchte ich einen Datenbank....
Gibt es da zu eine einfache Möglichkeit, ich habe davon noch keine Ahnung...
Vielen lieben Dank, falls jemand etwas dazu sagen kann...

**Comment by CommanderRedYT** [comment]
Hi, I wrote a guide here: https://github.com/tbnobody/OpenDTU-docs/pull/15/files
It did not get merged yet, but you can still check it out.
